src/http/operator.fixtures.ts

```diff
@@ -10,10 +10,12 @@ import { curatedModuleAddressWithCheckSum, dvtModuleAddressWithChecksum } from '
 
 export const dvtOperatorsResp: Operator[] = [operatorOneDvt, operatorTwoDvt].map((op) => ({
   ...op,
+  finalizedUsedSigningKeys: undefined,
   moduleAddress: dvtModuleAddressWithChecksum,
 }));
 
 export const curatedOperatorsResp: Operator[] = [operatorOneCurated, operatorTwoCurated].map((op) => ({
   ...op,
+  finalizedUsedSigningKeys: undefined,
```

Comment:
here should not be finalizedUsedSigningKeys in answer 
curatedOperatorsResp used for api answer check

Reply:
fixed

Reply:
you're right.
That's why I added that line.

Reply:
didnt get

Reply:
This is the code for preparing values for comparison in tests. I deleted this field because the endpoints don't give it out

Reply:
got it